src/rules_pygen/wheeltool.py

```diff
@@ -180,7 +180,15 @@ def dependencies(self, extra=None):
             for entry in requires:
                 # Strip off any trailing versioning data.
                 parts = re.split("[ ><=()]", entry)
-                yield parts[0]
+                package_name = parts[0]
+                # For some packages Requires-Dist contains extras defined like:
+                # tablib[html,ods,xls,xlsx,yaml] (>=0.14.0)
+                # Since we don't support extras, we should remove that part completely,
+                # otherwise we end up having `py_library.deps` like `tablib[html,ods,xls,xlsx,yaml]`
+                # which isn't a valid bazel dep
+                if package_name.endswith(']'):
```

Comment:
Seems ok, but would be better to match against /.+\[[^\]]+\]$/ or similar.

Reply:
I was thinking about using regex, but IMO `startswith` and `endswith` reduce the change to make a mistake